numpy/matlib.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 from numpy.matrixlib.defmatrix import matrix, asmatrix
-# need * as we're copying the numpy namespace (FIXME: this makes little sense)
-from numpy import *
+# need * as we're copying __all__ objects from the numpy namespace
+from numpy import *  # noqa: F403
```

Comment:
Why did we we remove the `FIXME` here?

Reply:
I was hoping to improve the comment to say why `from numpy import *` is needed in this context. Does this comment help, or can it be further improved?

Reply:
It seems this comment is further addressed in #15381, so I'm pulling this edit out of this PR.